tests/test_list.txt

```diff
@@ -10,6 +10,7 @@ examples/capture_old_request.py
 examples/capture_png.py
 examples/capture_to_buffer.py
 examples/capture_video.py
+examples/capture_video_multiple.py
```

Comment:
Might have squashed this one with the commit that defined that test, but not hugely bothered really!

Reply:
Will squash that down